src/CellContentParser.ts

```diff
@@ -127,11 +134,11 @@ export class CellContentParser {
     } else if (typeof content === 'boolean') {
       return new CellContent.Boolean(content)
     } else if (content instanceof Date) {
-      return new CellContent.Number(this.dateHelper.dateToNumber({
+      return new CellContent.Number(new DateNumber(this.dateHelper.dateToNumber({
         day: content.getDate(),
         month: content.getMonth() + 1,
         year: content.getFullYear()
-      }))
+      }), "Date()"))
```

Comment:
@wojciechczerniak not really sure how to solve it (what is the format of Date that came from Date() JS object)?

Reply:
I didn't noticed the comment, sorry. I think that if it was a Date object, we may serialize to Date object as well.

But I'm not sure how to handle the logic `= Date() + DD/MM/YY` 🤔

Reply:
The left operand aways wins in our logic and then its format is preserved.